hyperspy/models/edstemmodel.py

```diff
@@ -42,4 +42,5 @@ def __init__(self, spectrum,
                  auto_background=True,
                  auto_add_lines=True,
                  *args, **kwargs):
-        EDSModel.__init__(self, spectrum, auto_add_lines, *args, **kwargs)
+        EDSModel.__init__(self, spectrum, auto_background, auto_add_lines,
```

Comment:
Here you're actually fixing a bug that didn't cause any failure in the tests. It would be nice to add a little test for the `EDSModel` constructor.